channels-rke2.yaml

```
@@ -2521,6 +2521,26 @@ releases:
         repo: rancher-rke2-charts
         version: 0.1.2200
     featureVersions: *featureVersions-v1
+  - version: v1.29.14-rc1+rke2r1
+    minChannelServerVersion: v2.9.0-alpha1
+    maxChannelServerVersion: v2.10.99
+    serverArgs: *serverArgs-v1-28-9-rke2r1
+    agentArgs: *agentArgs-v1-28-8-rke2r1
+    charts: &charts-v1-29-14-rc1-rke2r1
+      rke2-multus:
+        repo: rancher-rke2-charts
+        version: v4.1.403
+      rke2-flannel:
+        repo: rancher-rke2-charts
+        version: v0.26.402
+      rancher-vsphere-csi:
+        repo: rancher-rke2-charts
+        version: 3.3.1-rancher900
+      rancher-vsphere-csi:
+        repo: rancher-rke2-charts
+        version: 3.3.1-rancher900
+      <<: *charts-v1-29-13-rke2r1
```

`<<: *charts-v1-29-13-rke2r1`  needs to go up at the TOP of the block, before defining any additional keys. This is why it is failing to find the correct versions of charts in this version.
```suggestion
    charts: &charts-v1-29-14-rc1-rke2r1
      <<: *charts-v1-29-13-rke2r1
      rke2-multus:
        repo: rancher-rke2-charts
        version: v4.1.403
      rke2-flannel:
        repo: rancher-rke2-charts
        version: v0.26.402
      rancher-vsphere-csi:
        repo: rancher-rke2-charts
        version: 3.3.1-rancher900
      rancher-vsphere-csi:
        repo: rancher-rke2-charts
        version: 3.3.1-rancher900
```